api/userRoutes/users.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const router  = express.Router();
+const User  = require("../models/models")
+const Parse = require('parse/node')
+
+// Get list of favorited cities
```

Comment:
the comment here says favorited cities - is this still the intention of the route? it just returns all users right now, correct?

Reply:
nice catch!